ui-tests/src/test/java/org/openmrs/reference/AddFormTest.java

```diff
@@ -18,22 +16,36 @@
  */
 public class AddFormTest extends TestBase {
     private HomePage homePage;
+    private static final By NEW_HTML_FORM = By.cssSelector("a[href='htmlForm.form']");
     private HeaderPage headerPage;
     private ManageFormsPage manageForm;
     private ClinicianFacingPatientDashboardPage patientDashboardPage;
+    private AdministrationPage administrationPage;
+    private ManageHtmlFormsPage manageHtmlFormsPage;
+    private HtmlFormsPage htmlFormsPage;
 
     @Before
     public void setUp() throws Exception {
         homePage = new HomePage(page);
-        assertPage(homePage);
         headerPage = new HeaderPage(driver);
+        administrationPage = new AdministrationPage(page);
         manageForm = new ManageFormsPage(driver);
         patientDashboardPage = new ClinicianFacingPatientDashboardPage(page);
+
     }
 
-    @Ignore //ignore due to moving forms functionality
     @Test
     public void addFormTest() throws Exception {
+        homePage.goToAdministration().clickOnManageHtmlForms().clickOnNewHtmlForm();
+        driver.manage().timeouts().implicitlyWait(0,TimeUnit.SECONDS);
+
+        HtmlFormsPage newHtmlForm = new HtmlFormsPage(page);
+        newHtmlForm.setDESCRIPTION("defining new form");
+        newHtmlForm.setNAME("newFormTest");
```

Comment:
merge all these methods into one and call just one method to create the form

Reply:
Resolved